src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -297,14 +320,20 @@ void convertRequireToImportStatements(
       return;
     }
 
-    // Local name is the shortened namespace symbol
-    String localName = nameUtil.lastStepOfName(fullLocalName);
-
     FileModule module = namespaceToModule.get(requiredNamespace);
 
     String moduleSuffix = nameUtil.lastStepOfName(requiredNamespace);
+
+    // Local name is the shortened namespace symbol
```

Comment:
I don't understand this comment. It is a good time to expand on the comments, maybe and example would help here.

Reply:
Expanded the comment and added a few examples.

`localNames` is not always local names, it's the last part of the namespace if the goog.require() is a side effect only import. Somehow that's consistent with the rest of the code.
Surprise :)

Reply:
wow, surprise indeed :)